pubspec.lock

```diff
@@ -7,21 +7,21 @@ packages:
       name: _fe_analyzer_shared
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "38.0.0"
+    version: "39.0.0"
   algolia:
     dependency: "direct main"
     description:
       name: algolia
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.0.5+1"
+    version: "1.1.0+2"
```

Comment:
Can we roll this back to v1.0.5 until they fix the web support? It doesn't seem that they added any new useful features.

Reply:
Or even better, seems like they just published v1.1.1 around an hour ago, supposedly fixing web support, but it hasn't been analyzed by pub yet.